mycroft/configuration/mycroft.ini

```diff
@@ -93,3 +93,10 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[PhillipsHueSkill]
+verbose = False
```

Comment:
Sweet. I'm fine with either as the default. Let me know if you, or anyone else, has a preference.